### ISSUE TITLE
How to add normalization layer for average power constraint?

### ISSUE THREAD

**Issue by tinyxuyan** [musicbeer/Deep-Learning-for-the-Physical-Layer#2]

**Comment by flysky6688** [comment]
Have you reslove this question? I also understand it.

**Comment by tinyxuyan** [comment]
Using keras.batchbormalization setting momentum=0, then you can get the result you want. With this method, I get the result of O'Shea's paper. Maybe there are other approaches.

**Comment by Cyprusky** [comment]
> Using keras.batchbormalization setting momentum=0, then you can get the result you want. With this method, I get the result of O'Shea's paper. Maybe there are other approaches.

Can you share your codes? It's very kind of you
